sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/SymbolicExecution/CSharp/CSharpExplodedGraph.cs

```diff
@@ -769,16 +769,17 @@ private void VisitCaseSwitchBinaryBranchBlock(BinaryBranchBlock branchBlock, Exp
         private void VisitSwitchExpressionArmBinaryBranch(BinaryBranchBlock branchBlock, ExplodedGraphNode node, SwitchExpressionArmSyntaxWrapper armSyntaxWrapper)
```

Comment:
One thing that I find a bit confusing now is that we handle the constant pattern differently than RecursivePattern. Constant pattern is processed already when building the CFG and it will appear as Identifier in the CFG but the RecursivePattern will remain as it is.

It's just a small thing and avoiding the additional changes keeps the PR small which is nice, but I had to mention it 😄